resource/ui/ts/util/querycache.ts

```diff
@@ -34,7 +34,7 @@ module Utils {
         constructor(private _query:() => promise<T>) {
             this.refresh()
         }
-            
+
```

Comment:
do we need to add some linting for typescript?
https://github.com/palantir/tslint